scripts/contrib/add_special_yaml_to_npz.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/python
```

Comment:
You really need a comment in this file saying what this is for and in which context it is to be used. Ideally with an example command line.

Reply:
I've added some commits which should address your request for changes. Please, let me know if it's ok now.